src/LazyOperations/InverseLinearMap.jl

```diff
@@ -0,0 +1,333 @@
+(M^T)^{-1}=(M^{-1})^T, it follows that ``σ(d, L) = M^{-1}⋅σ((M^T)^{-1} d, S)``
+for any direction ``d``.
+"""
+function σ(d::AbstractArray{N,1}, ilm::InverseLinearMap{N,S,NM,MAT}) where {N<:Real, S<:LazySet{N}, NM, MAT}
+    return ilm.M \ σ(transpose(ilm.M) \ d, ilm.X) #TODO: Check this formula
+end
+
+"""
+    ρ(d::AbstractVector{N}, ilm::InverseLinearMap{N}; kwargs...) where {N<:Real}
+
+Return the support function of the inverse linear map.
+
+### Input
+
+- `d`      -- direction
+- `ilm`    -- inverse linear map
+- `kwargs` -- additional arguments that are passed to the support function
+              algorithm
+
+### Output
+
+The support function in the given direction.
+If the direction has norm zero, the result depends on the wrapped set.
+
+### Notes
+
+If ``L = M^{-1}⋅S``, where ``M`` is a matrix and ``S`` is a convex set, it follows
+that ``ρ(d, L) = ρ((M^T)^{-1} d, S)`` for any direction ``d``.
+"""
+function ρ(d::AbstractVector, lm::InverseLinearMap)
+    return ρ(transpose(lm.M) \ d, lm.X)
+end
+
+
+"""
+    ∈(x::AbstractVector{N}, ilm::InverseLinearMap{N}) where {N<:Real}
+
+Check whether a given point is contained in an inverse linear map of a convex set.
+
+### Input
+
+- `x`   -- point/vector
+- `ilm` -- inverse linear map of a convex set
+
+### Output
+
+`true` iff ``x ∈ ilm``.
+
+### Algorithm
+
+This implementation does not explicitly invert the matrix since it uses the
+property ``x ∈ M^{-1}⋅S`` iff ``M⋅x ∈ S``..
+
+### Examples
+
+```jldoctest
+julia> ilm = LinearMap([0.5 0.0; 0.0 -0.5], BallInf([0., 0.], 1.));
+
+julia> [1.0, 1.0] ∈ ilm
+false
+julia> [0.1, 0.1] ∈ ilm
+true
+```
+"""
+function ∈(x::AbstractVector, lm::InverseLinearMap)
+    return lm.M * x ∈ lm.X
+end
+
+"""
+    an_element(ilm::InverseLinearMap{N})::Vector{N} where {N<:Real}
+
+Return some element of an inverse linear map.
+
+### Input
+
+- `ilm` -- inverse linear map
+
+### Output
+
+An element in the inverse linear map.
+It relies on the `an_element` function of the wrapped set.
+"""
+function an_element(lm::InverseLinearMap)
+    return lm.M \ an_element(lm.X)
+end
+
+"""
+    vertices_list(ilm::InverseLinearMap{N}; prune::Bool=true)::Vector{Vector{N}} where {N<:Real}
+
+Return the list of vertices of a (polyhedral) inverse linear map.
+
+### Input
+
+- `ilm`   -- inverse linear map
+- `prune` -- (optional, default: `true`) if true removes redundant vertices
+
+### Output
+
+A list of vertices.
+
+### Algorithm
+
+We assume that the underlying set `X` is polyhedral.
+Then the result is just the inverse linear map applied to the vertices of `X`.
+"""
+function vertices_list(ilm::InverseLinearMap{N}; prune::Bool=true) where {N}
+    # for a zero map, the result is just the list containing the origin
+    if iszero(ilm.M)
+        return [zeros(N, dim(ilm))]
+    end
+
+    # collect low-dimensional vertices lists
+    vlist_X = vertices_list(ilm.X)
+
+    # create resulting vertices list
+    vlist = Vector{Vector{N}}()
```

Comment:
`Vector{Vector{N}}()` works but is a bad idea for library code because the vertices of `X` don't need to be stored as regular arrays . better if we use the type of the vertices of `X` i guess that's `eltype(vlist_X)`.

Reply:
OK. I will test this

Reply:
```sizehint!``` needs a collection as the first entry. Then ```vlist = eltype(vlist_X)``` doesn't work.
I fixed ```vlist``` definition to:
```vlist = Vector{eltype(vlist_X)}();```
Then, I think we are at the same point than before...

Related to this function: does it need ```Polyhedra```?

Reply:
> vlist = Vector{eltype(vlist_X)}();

that's precisely what i meant above by " use the type of the vertices of X": `vlist` is a vector of vectors. it is ok to use the outer `Vector`  for storage.